aaaaAAAA/game.py

```diff
@@ -0,0 +1,110 @@
+import random
+
+import arcade
+
+from aaaaAAAA import _sprites, constants
+
+
+class GameView(arcade.View):
+    """
+    Main application class.
+
+    NOTE: Go ahead and delete the methods you don't need.
```

Comment:
Can we just yeet this note haha

Reply:
Let's do that when we have the actual game lol